bcrhp/src/bcrhp/schema/SiteClassificationSchema.ts

```diff
@@ -0,0 +1,58 @@
+import { z } from 'zod';
+
+const SiteClassificationSchema = z.object({
+    contributingResources: z
+        .string()
+        .min(1, { message: 'Number of Contributing Resources is required.' })
+        .max(250),
+    heritageClasses: z.array(z.string()).max(5),
+    functionCategory: z
+        .string()
+        .min(1, { message: 'Function Category is required.' })
+        .max(250),
+    heritageFunctions: z.array(z.string()).max(5),
+    heritageTheme: z
+        .string()
+        .min(1, { message: 'Heritage Theme is required.' })
+        .max(250),
+    heritageThemes: z.array(z.string()).max(5),
+});
+
+const requiredSiteClassificationSchema = SiteClassificationSchema.partial({});
+// @ts-ignore
+type SiteClassificationType = z.infer<typeof SiteClassificationSchema>;
+
+function getSiteClassificationSchema(): SiteClassificationType {
+    return new SiteClassification();
+}
+
+// @todo - Figure out object state - New/Updated/Deleted
+class SiteClassification implements SiteClassificationType {
+    constructor() {
+        this.contributingResources = '';
+        this.heritageClasses = [];
+        this.heritageCategory = '';
+        this.ownership = '';
+        this.functionCategory = '';
+        this.functionCategoryType = '';
+        this.heritageFunctions = [];
+        this.heritageTheme = '';
+        this.heritageThemes = [];
+    }
+    contributingResources: string;
```

Comment:
I think the grouping here isn't quite right. The values should be grouped the way they are grouped by the fieldsets eg 
1. \# of contributing resources, Heritage Category (should only be single pick - radio buttons not checkboxes), Ownership (UUID->Object map of these objects)
2. functionCategory, functionPeriod (UUID->Object map of these objects)
3. Heritage Themes (array)